### DISCUSSION TITLE
Airtasker for Everon Life

### DISCUSSION THREAD

**Discussion by SaxW** [EveronLife/EveronLife#125] [Ideas]
Had an idea which could add a cool dynamic to how the mod is played... 

What if you didn't have enough money to buy a truck and wanted to go on a concrete or heroin run? 
Or if you needed to collect multiple resources quickly but are playing solo?

Would be awesome if we added an [Airtasker](https://www.airtasker.com/au/?rfr=t) like app to the mod where you can hire people to do jobs for you. They would be able to complete tasks for you and get a clip of the payment.

**Comment by Arkensor** [comment]
I mean isn't this something people should do in-game via RP? Talk to people at the local market center and ask them if they want to earn some quick money?

**Comment by jasper-one** [comment]
I think a job board people could post on is a very cool concept rather than just the usual side chat or spamming questions in the main city  voice chat Kavala style.

A place where people could post "Job Ads" and people could accept them would be pretty cool. Very late game feature, but damn cool. Some people could probably make their money via the Job Board by picking up the different jobs people have to offer.

Examples of jobs could include: 

- Bounties 
- Armed Escort Services
- Vehicle Hire
- Crafting Services 
- And anything players can come up with

This could also include something like a Cash On Delivery (COD) system like World of Warcraft has where players can send goods to people for proposed prices and they can buy it from a mailbox of sorts. So someone could post "Buying Lumber, $100 a log" and you could send it to them so they can purchase it off you next time they are online". I like the idea of a properly async economy that can keep moving and not require everyone to be online for transactions to take place.

**Comment by Pyth3rEx** [comment]
To add some layering to this we could imagine a website template. Depending on how well enfusion is willing to connect to the outside world, ads posted in game could show on a webpage hosted by the server owner.

This would also allow for a lot of cross-app things to be done either by the base EL team or by server owners.

Linked to #202
